db/migrations/versions/222a1c3b6321_.py

```diff
@@ -0,0 +1,78 @@
+"""empty message
+
+Revision ID: 222a1c3b6321
+Revises: 55dcd4abf66a
+Create Date: 2023-08-08 11:13:18.177809
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "222a1c3b6321"
+down_revision = "55dcd4abf66a"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_table("flags")
+
```

Comment:
```suggestion
op.execute("DROP TYPE flagtype")
```

`flagtype` is not automatically dropped. This line should fix the previously mentioned error

Reply:
thanks @RamuniN